lib/src/model/movies_model.dart

```diff
@@ -11,15 +11,18 @@ class MoviesModel {
     _totalResults = parsedJson['total_results'];
     _totalPages = parsedJson['total_pages'];
 
-    List<MoviesResultModel> temp = [];
+    List<MoviesResultModel> temporalMovies = [];
```

Comment:
Why do you call this temporal movies?

Reply:
I supposed that list works only temporarily in that function to update the _results value, however I changed it